google-api-client/src/test/java/com/google/api/client/googleapis/batch/BatchRequestTest.java

```diff
@@ -708,10 +712,12 @@ public LowLevelHttpRequest buildRequest(String method, String url) {
 
           @Override
           public LowLevelHttpResponse execute() throws IOException {
-            assertEquals("multipart/mixed; boundary=__END_OF_PART__", getContentType());
+            assertTrue(getContentType().startsWith("multipart/mixed; boundary=__END_OF_PART__"));
             ByteArrayOutputStream out = new ByteArrayOutputStream();
             getStreamingContent().writeTo(out);
-            assertEquals(expectedOutput, out.toString("UTF-8"));
+            String actual = out.toString("UTF-8");
+            assertTrue(actual + "\n does not contain \n" + part1, actual.contains(part1));
+            assertTrue(actual.contains(part2));
```

Comment:
Would you add source code comment what is in-between part1 and part2?

Reply:
done